AlgebraInLean/World0LeanIntroSOLUTIONS/Sheet2Solutions.lean

```diff
@@ -31,75 +33,79 @@ example:
 /-
 Let's start with a few tactics that you are already familiar with from the NNG:
 
-exact
-apply
-intro
+- exact
+- apply
+- intro
 
 To continue getting used to reading Lean and working with the Lean Infoview, let's do a few levels
-from the NNG. Don't forget that you can also use "rw" and "rfl"! Delete the "sorry"s and fill in the
+from the NNG. Don't forget that you can also use `rw` and `rfl`! Delete the `sorry`s and fill in the
 proofs.
 -/
 
-/-NNG Implication World Level 1-/
+/- NNG Implication World Level 1 -/
 example (x y z : Nat) (h1 : x + y = 37) (h2 : 3 * x + z = 42) : x + y = 37 := by
   exact h1
   done
 
-/-NNG Implication World Level 2-/
-/-hint: zero_add still works outside of the NNG, as it is a theorem proven in mathlib.-/
+/-
+NNG Implication World Level 2
+hint: zero_add still works outside of the NNG, as it is a theorem proven in Mathlib.
+-/
 example (y x : Nat) (h1 : 0 + x = 0 + y + 2) : x = y + 2 := by
   rw [zero_add] at h1
   rw [zero_add] at h1
   exact h1
   done
 
-/-NNG Implication World Level 3-/
+/- NNG Implication World Level 3 -/
 example (x y : Nat) (h1 : x = 37) (h2 : x = 37 → y = 42) : y = 42 := by
   apply h2 at h1
   exact h1
   done
 
-/-NNG Implication World Level 6-/
+/- NNG Implication World Level 6 -/
 example (x : Nat) : x = 37 → x = 37 := by
   intro h1
   exact h1
   done
 
-/-Let's move on beyond the NNG to do some exercises you haven't seen before. The following exercises
-are from Kevin Buzzard's Formalizing Mathematics.-/
+/-
+Let's move on beyond the NNG to do some exercises you haven't seen before. The following exercises
+are from Kevin Buzzard's Formalizing Mathematics.
+-/
 
 variable (P Q R S T : Prop)
-/-Note that the variables we are working with here are of the type Prop.-/
+/- Note that the variables we are working with here are of the type Prop. -/
```

Comment:
Change these to one-line (`--`) comments?

Reply:
Why single-line? Although I do think that the comment should overall be changed to better explain what `variable` does.